packages/firestore/src/remote/serializer.ts

```diff
@@ -852,6 +853,72 @@ export function toQueryTarget(
   return result;
 }
 
+export function toRunAggregationQueryRequest(
+  serializer: JsonProtoSerializer,
+  target: Target
+): ProtoRunAggregationQueryRequest {
+  // Dissect the path into parent, collectionId, and optional key filter.
+  const result: ProtoRunAggregationQueryRequest = {
+    structuredAggregationQuery: {
+      aggregations: [
```

Comment:
hard code the count field here for now. should be updated down the road when we added aggregateFields to dynamically run aggregate query (count, sum, min....)